app/validators/date_within_boundaries_validator.rb

```diff
@@ -1,5 +1,5 @@
 class DateWithinBoundariesValidator < ActiveModel::EachValidator
-  MIN = 17
+  MIN = 27
```

Comment:
is this perhaps an opportunity to make these `MIN | MAX` terms a little clearer?

In reality I think they are both maximums -- in different directions -- and that the use of "min" (especially a MIN which is larger than the MAX) is inherently confusing. If we were to change these constants to allow us to say something like:

```rb
unless value.between?(MAX_YEARS_AGO.years.ago, MAX_YEARS_FROM_NOW.years.from_now)
```

would that be an improvement?

Feel free to ignore if you think it's not worth the candle.

Reply:
Thanks for this Ed, and yes I do agree. I'll make this tweak as part of this PR.